win32ss/gdi/ntgdi/freetype.c

```diff
@@ -4390,6 +4461,13 @@ FindBestFontFromList(FONTOBJ **FontObj, ULONG *MatchPenalty,
         ASSERT(FontGDI);
         Face = FontGDI->SharedFace->Face;
 
+        if (FontGDI->Magic != 0xDEADBEEF)
+        {
+            IntLockFreeType();
+            IntRequestFontSize(NULL, FontGDI, LogFont->lfWidth, LogFont->lfHeight);
+            IntUnLockFreeType();
+        }
```

Comment:
It looks to me that this initial call to IntRequestFontSize() should be done when a font is added in this linked list (so, before that FindBestFontFromList() function is called).

Reply:
No, I think font realization should be done after insertion.

Reply:
Initialize with the default value and resize again on font realization.

Reply:
OK.